### DISCUSSION TITLE
Possible to buy a PCB witch is Ready ?

### DISCUSSION THREAD

**Discussion by Flohon3** [OMOTE-Community/OMOTE-Firmware#6] [Q&A]
Hello, is it possible to buy a ready-made board, since I'm really clueless. I would like to try the other things myself.

Greetz

**Comment by taintedkernel** [comment]
I would be interested in one as well if this is possible.  If not, I'm considering ordering a PCB and the parts myself though I'm still new to custom PCBs and SMD work this advanced.

@CoretechR did you order your PCB's with ENIG finish (due to the ESP32 BGA), along with a stencil and use hotplate assembly?

Edit: I see now that the ESP32 is actually QFN, but has thermal vias which are what I noticed on the PCB.  Don't know if that implies anything different for the surface finish.

**Comment by CoretechR** [comment]
Thank you for your interest in this project, I really appreciate it. But right now I don't have plans to sell boards or kits. 
The project needs more polishing and work especially in terms of software. 
I will update you if anything changes in that regard.

@taintedkernel I ordered the PCBs with HASL but only due to price difference. Both surfaces should be fine for this PCB. And yes, solder paste and a hot plate or oven is necessary. I have a little MHP30 hot plate that you can see in the YouTube video.

**Reply by mrjackson** [reply]
You could do something like PCBWay's Shared Projects. It lets you upload the design and then anybody that wants to can order their own boards. It makes it easier for the people that want to build but do not have the knowledge of gerber files and the like.

**Reply by CoretechR** [reply]
@mrjackson That sound interesting, I will definitely look into it. Do you know if they can populate the PCB as well? I really like the idea of selling board to people but assembling them myself is out of the question. It takes me like 2 hours to populate and solder one board.

**Reply by mrjackson** [reply]
I belive they do offer at least surface mount assembly. JLPCB does offer assembly also, but I'm not sure if they offer a shared project like service. I'm sure there's other similar options out there also.

**Reply by riker09** [reply]
Maybe you can get in touch with other makers that have successfully launched an project that required assembling a custom PCB? I'm not sure, but have you heard about Bangle.js? It's a hackable smartwatch that can be ordered fully assembled. Or maybe the people from the famous frame.work laptop might be willing to help?

I'm neither representative or a business person, but it feels to me that *a lot* of people are looking for a way to replace or upgrade their (smart home) controls but are afraid of vendor lock in or miss something in the available solutions.

**Comment by TomW80** [comment]
I am also interested in a finished board ;)

**Comment by ramonflores08** [comment]
I just ordered the pcbs and components to make one. I am currently printing the files and am excited to make a few of these remotes in case i lose one

**Comment by Dark1886** [comment]
I will also have about 5 boards worth of material soon. I would be willing to sell, but it wouldn't be the cheapest since I did have them pre-populated.

**Reply by taintedkernel** [reply]
@Dark1886 If you get them and verify they are functional, I would be interested in one.  I've had nothing but an absolute nightmare trying to assemble mine without any luck.

**Reply by Dark1886** [reply]
I should have a few more spares. I will try and send one out, but that probably wouldn't be for at least another week.

**Reply by thijstriemstra** [reply]
i'd be interested in one as well! 😀

**Reply by tsoluti** [reply]
For how much i can get a fully populated one?

**Comment by ghost** [comment]
[JLCPCB offers assembly](https://jlcpcb.com/smt-assembly) when providing GERBER, BOM&CPL file. Both sides assembly seems to be very expensive, but one side seems to be in a fair price range. And the side with the buttons does not seem to be hard to solder (in contrast to the other side^^). Will take a deeper look tomorrow.

Also it seems that 5 is the minimal available order quantity.I will order from Germany, Ruhrgebiet. If someone is interested in one and also located in Germany I could send it as soon as the 5-pack arrives. Surely less expensive then everybody ordering five of them^^.

**Reply by DrRSatzteil** [reply]
Would be interested in the price. I might take one if it seems reasonable ☺️

**Reply by TomW80** [reply]
For me the same if price fits

**Reply by DrRSatzteil** [reply]
I'm still afraid of the soldering however 🙈

**Reply by thijstriemstra** [reply]
I'm in the Netherlands and would be interested as well @Insomniac80 any idea on the price?

**Reply by ghost** [reply]
Sorry for the late reply, I am actually sick and feel very bad. As soon as it goes better I will take a look and reply here. 
First look I have taken showed less then 20€ for 5 pcbs including (slow) shipping cost. Have to create an account and see what the components will cost including the assembly.

**Reply by Westworld** [reply]
(from Germany): I would be interested (one/difficult side soldered)

**Reply by juli-dot-md** [reply]
(Germany) Also interested

**Reply by GaryTill** [reply]
I've had the blank PCBs and most of the components for months now but full time work, then home to cook, clean and sleep have prevented me from looking at the project.

I'd definitely be up for buying an assembled PCB.

Also if anyone in Australia wants PCBs, hit me up I have 10 spare.

**Reply by DKeppi** [reply]
Im also interested (Austria)
3D Printing i could do on my own but PCB assembling is not my case

**Comment by skiphansen** [comment]
Still very interested in 2024!  My logitech universal remote finally died and I'm back to a table full of remotes none of which do I like or know how to use.  This is AMAZING I want one !!!  I'm afraid it's past my soldering skills and I don't have a 3d printer.

**Comment by skiphansen** [comment]
Anyone interested should head over to discord where a potential group buy is being organized.

**Comment by Delgrey** [comment]
Is the Discord channel still running in 2025?

**Comment by skiphansen** [comment]
Yes very much so and there's a [worldwide bulk order](https://discord.com/channels/1138116475559882852/1310200827943583775) about to go down.

**Reply by Delgrey** [reply]
How do I join the channel please. That link and all others just go to an error page in the web version of Discord saying there are no text channels. The mobile version just throws me into whatever channel I was when I last opened the app. I'm keen to get one or two boards in Australia.

**Reply by skiphansen** [reply]
Try the link from the projects readme: https://discord.com/channels/1138116475559882852/1153343867681243279
If that doesn't work I can't help you, I'm just a user like you.

**Reply by Delgrey** [reply]
That's the link I tried first. It's invalid. Just opens Discord and shows a random channel (in my case Home Assistant).

I think they changed the link format. Discord says:

"Invites should look like https://discord.gg/hTKzmak, hTKzmak, or https://discord.gg/wumpus-friends"

Not a really long numeric number like above.

Are you able to message that Admins on the Discord channel for me and ask for a valid link? I assume no one new can join.

**Reply by KlausMu** [reply]
Both links from README work for me:
First the invitation:
https://discord.gg/5PnYFAsKsG
Then the board-buy-sell
https://discord.com/channels/1138116475559882852/1153343867681243279

**Reply by Delgrey** [reply]
Thanks! Yes the GG one works. I only saw the .com one in the readme and assumed it was the main link. The .com one only works if you are already a channel member. (No error message. Thanks Discord!). Thanks for your help all. I'm in.

**Comment by SIR-Inc** [comment]
As an electronics engineer with a lot of experience in product development and understanding the logistics and reality of hardware development and production I feel I need to warn people without technical knowledge and or experience not to attempt the DIY approach and order the boards and components with the intention of trying to assemble at home. A recent video posted by [BeardedTinker](https://www.youtube.com/watch?v=lom9cs3mI4A) on his YouTube channel does a very job of explaining the process for both options and I agree with his recommendation of ordering the fully assembled boards.

**Reply by taintedkernel** [reply]
I would have to agree with this.  I have some experience with electronics and tried sourcing the parts, assembling one by hand, and it didn't end up functioning.  I've tried debugging without much success, so I went in to the recent batch order on the Discord and I'm looking forward to when they are getting sent out.
